src/meson.build

```diff
@@ -0,0 +1,37 @@
+subdir('protocol')
+
+srcs = files(
+	'gtk-layer-shell.c',
+	'gtk-wayland.c',
+	'custom-shell-surface.c',
+	'layer-surface.c',
+	'xdg-popup-surface.c',
+	'xdg-toplevel-surface.c',
+	'gdk-window-hack.c',
+	'simple-conversions.c',
+)
+
+lsh_lib = both_libraries('gtk-layer-shell',
```

Comment:
Can we name this something longer than `lsh_lib`? `gtk_layer_shell_lib` doesn't seem like too much typing to me.